tests/scripts/minikube.sh

```diff
@@ -82,6 +83,11 @@ case "${1:-}" in
     copy_image_to_cluster mysql:5.6 mysql:5.6
     copy_image_to_cluster wordpress:4.6.1-apache wordpress:4.6.1-apache
     ;;
+  helm)
+    echo " copying rook image for helm"
+    helm_tag="`cat _output/version`"
+    copy_image_to_cluster ${BUILD_REGISTRY}/rook-amd64 rook/rook:${helm_tag}
```

Comment:
would it make sense to use `master` for the tag in the dev environment? This would follow the same pattern as the `up` and `update` commands that copy the rook and toolbox containers